librz/sign/create.c

```diff
@@ -87,14 +98,15 @@ static RzFlirtModule *flirt_module_new(RzAnalysis *analysis, RzAnalysisFunction
 
 	if (b_size > 0 && buffer) {
 		// the crc should be generated only for when the buffer is > RZ_FLIRT_MAX_PRELUDE_SIZE
-		module->crc_length = RZ_MIN(b_size, 0xFF);
+		// also the size can be zero if after the prelude there is only masked bytes
+		module->crc_length = flirt_crc16_length(mask, b_size);
```

Comment:
shouldn't we consider just `mask+RZ_FLIRT_MAX_PRELUDE_SIZE`?

Reply:
no, mask is already shifted by 32 bytes

Reply:
https://github.com/rizinorg/rizin/blob/8caaad7d17a7448d465ecc8530ec97b2360cf425/librz/sign/create.c#L198